app/src/main/res/values/strings.xml

```diff
@@ -25,19 +25,19 @@
     <string name="accept">Accept</string>
     <string name="reject">Reject</string>
     <string name="stop">Stop</string>
-    <string name="sending">Sending</string>
+    <string name="sending">Sending…</string>
```

Comment:
This will not look as nice.

![Screenshot_20210829-081517_TShot_DEBUG_2](https://user-images.githubusercontent.com/10258162/131239371-ccfc7043-d3c1-4960-8611-66657119bbcc.png)

Reply:
I don't understand. It isn't sending just 26%…? Presumably it has sent 26%.
Ideally the … should be after the 26%. "Sending, 26%…"
Then "Ongoing Task" can be dropped.

Reply:
```suggestion
    <string name="sending">Sending</string>
```